packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -47,6 +47,7 @@ export const styles = css`
                     Not sure why but this is needed to get buttons with icons and buttons
                     without icons to align on the same line when the button is inline-flex
                     See: https://github.com/microsoft/fast/issues/5414
+                    For additional discussion, see: https://github.com/ni/nimble/issues/503
```

Comment:
Don't need to link to both, just add useful info to our issue

```suggestion
                    See: https://github.com/ni/nimble/issues/503
```